File: C/retangulo.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

double quadrado(double n){
	return n*n;
}

int main(){
	double base, altura;
	printf("Base do retangulo: ");
	scanf("%lf", &base);
	printf("Altura do retangulo: ");
	scanf("%lf", &altura);
	
	printf("AREA = %.4lf\n", (altura*base));
	printf("PERIMETRO = %.4lf\n", (2*altura+2*base));
	printf("DIAGONAL = %.4lf\n", sqrt(quadrado(altura)+quadrado(base)));
	
	
	
	return 0;
}

File: C#/retangulo.cs
using System;
using System.Globalization;

class Retangulo {
	static void Main(string[] args){
		CultureInfo CI = CultureInfo.InvariantCulture;
		
		double baseR, altura;
		Console.Write("Base do retangulo: ");
		baseR = double.Parse(Console.ReadLine(),CI);
		Console.Write("Altura do retangulo: ");
		altura = double.Parse(Console.ReadLine(),CI);
		
		
		Console.WriteLine("AREA = " + (baseR*altura).ToString("F4",CI));
		Console.WriteLine("PERIMETRO = " + (2*(baseR+altura)).ToString("F4",CI));
		Console.WriteLine("DIAGONAL = " + (Math.Sqrt(baseR*baseR+altura*altura)).ToString("F4",CI));
		
	}
}